### PR TITLE
Scarf Changes

### PR DESCRIPTION
**Pull request by 1Sparky1**
Adds new sprites for scarves, which can be cycled between by using in hand. The zebra scarf is now also recolourable.

image

### CODE REVIEW

**Comment by 1Sparky1** [comment]
!review

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
please keep the current scarf sprite as an option.

**Review by SleepyGemmy — COMMENTED** [review]
i suggest utilizing the radial menu system for selecting the sprites.

**Comment by 1Sparky1** [comment]
!wip

**Comment by 1Sparky1** [comment]
!review

**Comment by 1Sparky1** [comment]
Old sprites are back and switched the style change to a radial menu.

File: code/modules/clothing/under/accessories/accessory.dm [hunk 1]
```diff
@@ -314,16 +314,44 @@
...
-	flippable = 1
+	icon = 'icons/obj/clothing/scarves.dmi'
+	icon_state = "scarf0"
+	item_state = "scarf0"
+	contained_sprite = TRUE
+	var/list/alternatives = list("drape" = "scarf0",
+								"drape, flipped" = "scarf0_f",
+								"single wrap" = "scarf1",
+								"double wrap" = "scarf2",
+								"parisian" = "scarf3")
```
Comment (SleepyGemmy) [inline code/modules/clothing/under/accessories/accessory.dm:325]:
```suggestion
	var/list/alternatives = list(
		"drape" = "scarf0",
		"drape, flipped" = "scarf0_f",
		"single wrap" = "scarf1",
		"double wrap" = "scarf2",
		"parisian" = "scarf3"
	)
```

File: code/modules/clothing/under/accessories/accessory.dm [hunk 2]
```diff
@@ -314,16 +314,44 @@
...
+	var/list/overlay_alternatives = null
+
+/obj/item/clothing/accessory/scarf/attack_self(mob/user)
+	if(alternatives)
+		var/list/options = list()
+		for (var/i in alternatives)
```
Comment (SleepyGemmy) [inline code/modules/clothing/under/accessories/accessory.dm:331]:
```suggestion
		for(var/i in alternatives)
```

**Review by SleepyGemmy — APPROVED** [review]

**Review by SleepyGemmy — APPROVED** [review]
DMIs look fine after the master merge.

**Pull request merged**